### PR TITLE
#1070

### PR DESCRIPTION
**Pull request by wsbrenk**
1. changes proposed in this pull request:
 
   - fixes issue #1070
   
   - if not fixing an existing issue, comments explaining the purpose of the PR should be provided)
 
 
2. changelog and release_notes ...

 - [ ] have been updated
 - [x] do not require update

3. [Optional] suggested person to review this PR @___

### CODE REVIEW

File: src/main/java/core/model/enums/RatingsStatistics.java [hunk 1]
```diff
@@ -21,7 +16,7 @@
     public int getValue() {
         return value;
     }
-
+/*
```
Comment (tychobrailleur) [inline src/main/java/core/model/enums/RatingsStatistics.java:19]: Just remove if this isn't needed any more.

File: src/main/java/module/series/Model.java [hunk 2]
```diff
@@ -1,14 +1,44 @@
...
+			sum += v;
+			if (v > max) max = v;
+		}
+
+		public int getMax() {
+			return max;
+		}
+
+		public int getAverage() {
+			if (count > 0) return (int) ((double) sum / count + .5);
```
Comment (tychobrailleur) [inline src/main/java/module/series/Model.java:34]: What is the `+ .5` for here?  looks hackish ;-)

File: src/main/java/module/series/SeriesPanel.java [hunk 3]
```diff
@@ -28,15 +30,15 @@
 	private MatchDayPanel[] matchDayPanels;
 	private SeriesHistoryPanel seriesHistoryPanel;
 	private Model model;
-	private Map<Integer, Map<RatingsStatistics,Integer>> leagueStatistics;
+	//private Map<Integer, Map<RatingsStatistics, Model.StatisticsEntry>> leagueStatistics;	// team id -> (statistic key -> value)
```
Comment (tychobrailleur) [inline src/main/java/module/series/SeriesPanel.java:33]: just remove commented out stuff from this class.

File: src/main/java/module/series/Model.java (hunk 2 above)
Comment (wsbrenk) [reply src/main/java/module/series/Model.java:34]: this is how the c++ developer rounds off

File: src/main/java/core/model/enums/RatingsStatistics.java (hunk 1 above)
Comment (wsbrenk) [reply src/main/java/core/model/enums/RatingsStatistics.java]: removed it

File: src/main/java/module/series/SeriesPanel.java (hunk 3 above)
Comment (wsbrenk) [reply src/main/java/module/series/SeriesPanel.java]: removed

File: src/main/java/module/series/Model.java (hunk 2 above)
Comment (wsbrenk) [reply src/main/java/module/series/Model.java]: changed to Math.round

**Pull request merged**